tasks/steps/blocks/unpack_tgz.yml

```
@@ -0,0 +1,60 @@
+---
+- name: Create group
```

Let's use quotation marks in names
```suggestion
- name: 'Create group'
```